pkg/azurefile/azurefile.go

```diff
@@ -429,11 +429,7 @@ func (d *Driver) GetAccountInfo(volumeID string, secrets, reqContext map[string]
 				diskName = v
 			}
 		}
-		if fileShareName != "" {
-			accountName, accountKey, err = getStorageAccount(secrets)
-		} else {
-			err = fmt.Errorf("could not find sharename from context(%v)", reqContext)
-		}
+		accountName, accountKey, err = getStorageAccount(secrets)
```

Comment:
Why isn't the judgement required? If the fileShareName isn't specified, the return value is "". Then we can not get the fileURL in `ControllerPublishVolume` and `ControllerUnpublishVolume`.

Reply:
we want to make `getStorageAccount` as simple as possible, and if `fileShareName` is required, then the invoker should check whether `fileShareName`. And for this case, `ControllerPublishVolume` and `ControllerUnpublishVolume` does not need `fileShareName`